index.js

```diff
@@ -1,88 +1,141 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
+counter = 1; 
+counter = 2; 
```

Comment:
Although this will work, it's best practice to declare your variables with a keyword.
```suggestion
let counter = 1; 
counter = 2; 
```